velox/exec/Task.h

```diff
@@ -29,7 +29,7 @@ namespace facebook::velox::exec {
 class PartitionedOutputBufferManager;
 
 class HashJoinBridge;
-class CrossJoinBridge;
+class NestedLoopJoinBridge;
```

Comment:
To help further reduce the size of this PR, consider extracting renaming of CrossJoinProbe/Build/Bridge into a separate PR.

Reply:
#4678 is for renaming, thanks.